apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/disable-core-nux.js

```diff
@@ -21,6 +21,9 @@ subscribe( () => {
 	}
 	if ( select( 'core/edit-post' )?.isFeatureActive( 'welcomeGuide' ) ) {
 		dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' );
-		dispatch( 'automattic/nux' ).setWpcomNuxStatus( { isNuxEnabled: true } );
+		dispatch( 'automattic/nux' ).setWpcomNuxStatus( {
+			isNuxEnabled: true,
+		} );
+		dispatch( 'automattic/nux' ).setTourOpenStatus( { isTourManuallyOpened: true } );
```

Comment:
Rather than dispatching to actions to the same store maybe it makes sense to coming them?

```typescript
 		dispatch( 'automattic/nux' ).setWpcomNuxStatus( {
 			isNuxEnabled: true,
 			tourWasOpenedManually: true,
 		} );
```

It made me think though, is there ever a time when `isNuxEnabled` is set to `true` and it wasn't a manual opening? In which case why's the extra flag needed? 🤔

Reply:
I must just be confused, because it's working in manual testing

Reply:
> It made me think though, is there ever a time when isNuxEnabled is set to true and it wasn't a manual opening? In which case why's the extra flag needed? 🤔

For a brand new user, `isNuxEnabled = true` and the WelcomeGuide (Tour or NUX) opens automatically when the user opens the editor for the first time.  So yes, that is the time. :) That is the scenario we want to capture the "First time" viewing.  
After user dismisses the Guide, isNuxEnabled is set to false.   
For me, what has been a source of confusion is that this new-user-flag, which is a user setting, is then used a toggle when user accesses the More Menu.  I'd like to clean that logic up and use terminology similar to GB [isFeatureActive](https://github.com/WordPress/gutenberg/blob/c88866cd91ea3eb7990a68978e03e2366ed7106c/packages/edit-post/src/components/welcome-guide/index.js#L22-L23).    This isn't important for MVP though, so going on my list of todos.

Reply:
> Rather than dispatching to actions to the same store maybe it makes sense to combining them?

Ramon suggested the same.  I decided not to do that because of a reason that isn't coming to mind now! Maybe just clarity and because the 'nux' terminology made the logic harder for me to understand.  (Because I think the open / manual toggles are about the state of feature, not the state of the user.)